beacon-chain/sync/rpc.go

```diff
@@ -17,6 +21,10 @@ import (
 // they don't receive the first byte within 5 seconds.
 var ttfbTimeout = 5 * time.Second
 
+// maxChunkSize would be the maximum allowed size that a request/response chunk can be.
+// any size beyond that would be rejected and the corresponding stream reset.
+var maxChunkSize = 1 << 20
```

Comment:
Can you add a comment on what this size is in human readable terms? 2Mb? How big is `1 << 20`?